### ISSUE TITLE
Having issues scanning QR codes on iOS using 6.0.2

### ISSUE THREAD

**Issue by slonovost** [phonegap/phonegap-plugin-barcodescanner#333]
Having issues scanning QR codes on iOS.  In order for the plugin to detect the barcode, I'm forced to tilt or even rotate my device upside down to get it to scan.

Please find one of the QR codes having this issue attached.

Has anyone seen this before?  I'm running on iOS 9.3.5, but it's also reproducible on iOS 10.

<img width="345" alt="screen shot 2016-09-27 at 3 33 27 pm" src="https://cloud.githubusercontent.com/assets/22484656/18894602/c4c32252-84c8-11e6-9551-c61d6e929c61.png

**Comment by grmorcl** [comment]
I've noticed the same issue on both iOS and Android with QR codes - need to rotate my device 90 degrees to read the QR code you posted.

I've also noticed on both iOS and Android that it doesn't scan barcodes!?  Will raise a separate issue about that.

**Comment by KixS** [comment]
I have as well this issue for iOS.

When I try to scan a QRcode, even a relatively simple one, the scanner has trouble reading it. I have to move my phone forward and backward so it can capture at some point the QRcode, sometimes it can take up to 30s. This is with version 6.0.2 of the plugin.

Using the 3.0.1 -> it scans the QRcode almost in an instant.

**Comment by EddyVerbruggen** [comment]
Can you try again with 6.0.4? There's been a change where we hint the camera to look for objects near the camera (as opposed to anywhere).

**Comment by KixS** [comment]
Hi Eddy,

Thanks for replying.
I tried with 6.0.4, and it didn't improve the scan on iOS. I tried with 2 devices, a iPhone5S and a iPhone6S and on both there is a meaningful difference between 6.0.4 and 3.0.1. I am not sure how I can help but these are my observations for iPhone5S when the QRcode is almost matched to the green square (the QRcode is 130 characters):

- 3.0.1: almost instantaneous detection

- 3.1.0: still good detection, but slightly slower than 3.0.1. ~1s

- 4.0.1: ~3 to 5s detection time

- 5.0.1 onward: can't scan the QRcode with the camera so close, I need to pull away the camera and then it is sort of random when the QRcode is finally read. On average after 5/10s of manoeuvers.

On the iPhone6S, the detection is on general faster and I almost can't see a difference between 3.0.1. and 4.0.1 but the difference becomes noticeable from 5.0.1. where I can't scan on close distance again and need a bit of manoeuvers.
